.prettierrc

```diff
@@ -1,3 +1,8 @@
 {
-  "tabWidth": 4
-}
+  "arrowParens": "avoid",
+  "semi": false,
```

Comment:
We do this in a few projects, but I really hate it.. This is abusing js error correction, and the edge cases where prettier does it wrong is a time sink

Reply:
We should propagate this change to all repos then :)